smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftWriter.kt

```diff
@@ -106,6 +106,8 @@ class SwiftWriter(
 
     fun addImport(symbol: Symbol) {
         symbol.references.forEach { addImport(it.symbol) }
+        val additionalImports = symbol.getProperty("additionalImports").getOrElse { emptyList<Symbol>() } as List<Symbol>
+        additionalImports.forEach { addImport(it) }
```

Comment:
Additional imports were moved from the bottom to the top of this method because the early-exit on line 111 below causes additional imports to not be added.